pkg/pillar/cmd/zedagent/handleconfig.go

```diff
@@ -579,9 +575,17 @@ func getLatestConfig(getconfigCtx *getconfigContext, url string,
 				if config != nil {
 					log.Noticef("Using saved config dated %s",
 						ts.Format(time.RFC3339Nano))
+
+					cfgRetval := inhaleDeviceConfig(getconfigCtx, config, savedConfig)
+					if cfgRetval != configOK {
+						log.Errorf("inhaleDeviceConfig failed: %d", cfgRetval)
+						return cfgRetval, rv.TracedReqs
+					}
```

Comment:
Have you seen the inhale fail on a saved config? 
I don't have an issue with this added check, but wonder if there is some unknown issue which causes failures.

Reply:
Ah this is due to obsoleteConfig. Makes sense.